library_test.py

```diff
@@ -117,12 +117,21 @@ async def main() -> None:
     print("Devices:", devices)
     print("-" * 20)
     data = await api.get_sensor_data()
-    print("Data:", data)
+    print("Sensor Data:", data)
     print("-" * 20)
     print("Serial #:", data["sys_serial_number"])
     print("Firmware:", data["sys_firmware_version"])
     print("Hardware:", data["sys_hardware_version"])
     print("Uptime  :", api.convert_uptime(data["sys_uptime"]))
+    print("WAN status:", data.get("wan_status"))
+    print("Cable modem status:", data.get("cm_status"))
+    print("Cable modem mode:", data.get("cm_mode"))
```

Comment:
```suggestion
    print("LAN mode:", data.get("lan_mode"))
```

Reply:
It is related to above change. Further change to get it working.

Feel free to choice your wording.